package.json

```diff
@@ -8,10 +8,13 @@
   },
   "version": "0.2.0",
   "engine": {
-    "node": ">=0.10"
+    "node": ">=8.x"
```

Comment:
Current LTS is 6.11.3, I don't think we should require newer than that. New dependency `node-zabbix-sender` doesn't appear to require 8.x.